src/components/title/_fields/identifiers/identifiers-fields.js

```diff
@@ -13,96 +12,101 @@ import {
 
 import { FormattedMessage } from 'react-intl';
 
-export default class IdentifiersFields extends Component {
-  validateId(value) {
-    let errors;
+import { IDENTIFIERS_FIELDS_VALUE_MAX_LENGTH } from '../../../../constants';
 
-    if (!value) {
-      errors = <FormattedMessage id="ui-eholdings.validate.errors.identifiers.noBlank" />;
-    }
+const validateId = (value) => {
```

Comment:
Why you move this function outside the component?

Reply:
Because there's no consistency in code: in some components functions are outside, in some the are inside the component. It's fixed now and I'll keep this in mind for the future, thanks.